types/lint-staged/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for lint-staged 13.2
+// Project: https://github.com/okonet/lint-staged#readme
+// Definitions by: Andrey Okonetchnikov <https://github.com/okonet>
+//                 Saiichi Hashimoto <https://github.com/saiichihashimoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.5
```

Comment:
```suggestion
```
there is nothing specific in definition for TS version, just in tests, see other comment